### PR TITLE
Rebased as we discussed in Enhanced  UI PR 

### PR DESCRIPTION
**Pull request by damarudhvarma**
#### verify the changes and merge the PR with adding hachtoberfest labels .
Issue #13  resloved

### CODE REVIEW

File: src/CodeConverter.jsx [hunk 1]
```diff
@@ -59,6 +59,8 @@ const useShikiMonaco = () => {
   return { isReady, highlighter: highlighterRef.current };
 };
 
+import ClipLoader from "react-spinners/ClipLoader";
```
Comment (aayushai) [inline src/CodeConverter.jsx:62]: import Cliploader, famoon and fasun have already been done at start of this file with other imports.
remove line 62 and line 63

**Review by aayushai — CHANGES_REQUESTED** [review]
import Cliploader, famoon and fasun have already been done at start of this file with other imports.
remove line 62 and line 63, they are causing error during deployment

File: package.json [hunk 2]
```diff
@@ -13,15 +13,22 @@
...
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-icons": "^5.3.0",
     "react-spinners": "^0.14.1",
     "react-toastify": "^10.0.5",
     "shiki": "^1.22.0",
     "spinners": "^1.2.2",
-    "toastify-js": "^1.12.0"
+    "toastify-js": "^1.12.0",
+    "spinners": "^1.2.2",
```
Comment (aayushai) [inline package.json:29]: remove spinners from here, it is already there at line 27

**Comment by damarudhvarma** [comment]
@aayushai  once check now i've edited the files

**Comment by aayushai** [comment]
changes look good but i don't know why vercel bot is not able to deploy, it is still deploying the code before changes.

**Comment by aayushai** [comment]
Done good work.

**Comment by damarudhvarma** [comment]
> Done good work.

can please add hacktoberfest labels

**Pull request merged**